subprojects/packagefiles/utf8proc/meson.build

```diff
@@ -1,22 +1,19 @@
 project('utf8proc', 'c',
-    version : '3.2.1',
+    version : '2.7.0',
     license: 'MIT',
     default_options: ['c_std=c99']
 )
 
-utf8proc_args = []
-
-if get_option('default_library') == 'static'
-    utf8proc_args += '-DUTF8PROC_STATIC'
+if get_option('default_library') != 'static' and host_machine.system() == 'windows'
+  add_project_arguments('-DUTF8PROC_EXPORTS', language: 'c')
 endif
 
 utf8proc_lib = library('utf8proc',
-    include_directories: include_directories('.'),
-    c_args: ['-DUTF8PROC_EXPORTS']
+  'utf8proc.c',
+  include_directories: include_directories('.'),
```

Comment:
This is never needed. The current source and build dirs are in include dirs by default.